pom.xml

```diff
@@ -328,13 +327,6 @@
                     <propertyExpansion>main.basedir=${main.basedir}</propertyExpansion>
                     <excludes>**/module-info.java</excludes>
                 </configuration>
-                <dependencies>
```

Comment:
Why removed?

Reply:
No longer needed, if I read correctly new checkstyle config, this is a default set of rules (no need to duplicate explicitly)